### DISCUSSION TITLE
Cloudflare-   Record the visitor's real IP and not the one from Cloudflare

### DISCUSSION THREAD

**Discussion by zzsergio** [rejetto/hfs#735] [Q&A] [answered]
Hi.

In the hfs-2 version, the visitor's IP is recorded, not Cloudflare's.
But in this version 3 I have not been able to register the real IP of whoever visits my server.
I installed the reverse-proxy plugin, but it still logs cloudflare IP.

Do you know how to solve this problem?

Thanks..

PD. I don't speak English, translation done by Google. Apologies.

**Comment by rejetto** [comment, marked as answer]
google translation is fine :) I also speak italian by the way.
reverse-proxy plugin is NOT what you need, you can uninstall it.
that's only to make HFS a proxy, but in your case CloudFlare is the proxy.
I will try to improve its description so other people won't be fooled.

if you access admin-panel in you HFS you should see a warning that you are behind a proxy, and also tells you what to do. Can you confirm this please?

anyway, you just have to go to Admin-panel > Options, and set "Number of HTTP proxies" to 1.

**Comment by zzsergio** [comment]
Hi.

I only speak spanish.

> if you access admin-panel in you HFS you should see a warning that you are behind a proxy, and also tells you what to do. Can you confirm this please?

That information does not appear anywhere.

> anyway, you just have to go to Admin-panel > Options, and set "Number of HTTP proxies" to 1.

With this configuration the problem is solved.

Thanks..

Thank you very much for the great work done with HFS.
It is an excellent server.
